src/bundles/config.js

```diff
@@ -60,8 +60,7 @@ function getURLFromAddress (name, config) {
   try {
     const address = config.Addresses[name]
     return toUri(address).replace('tcp://', 'http://')
-  } catch (error) {
-    console.log(`Failed to get url from Addresses.${name}`, error)
+  } catch (_) {
```

Comment:
This smothers all errors, which we don't want to do. The error we can ignore is where `config` is undefined or null.